style.css

```diff
@@ -155,25 +164,30 @@ html {
 	justify-content: center;
 	font-family: 'Varela Round', sans-serif;
 	font-size: 3rem;
+	padding-top: 50px;
+	padding-bottom: 75px;
+	margin: 0;
 }
 
 .container {
 	display: flex;
 	align-items: center;
 	justify-content: center;
 	gap: 30px;
-	padding-top: 15px;
-	padding-bottom: 15px;
+	padding-top: 30px;
 	font-family: 'Varela Round', sans-serif;
 	font-weight: bold;
 	color: #2a2e32;
 	font-size: 2rem;
 	height: 40vh;
```

Comment:
I think this vh might also be causing some issues with your containers. There's no padding between sections when on mobile and the container goes under the title. The image also becomes very tiny. It'd be nice to see this container expand to fill more of the page and then the image could be bigger too
![image](https://user-images.githubusercontent.com/45392468/149966637-e4f5f1d6-f5e7-4a13-a112-279c2986751a.png)